content/en/docs/rules/custom-rules.md

```diff
@@ -0,0 +1,377 @@
+---
+
+To write a custom rule for Falco from scratch, it is essential to understand the conditions that need to be met for Falco to trigger an alert. However, this task is not simple as it requires considering the potential false positives and negatives that may arise from the rule.
+
+## Rules Placement
+
+When adding a new rule to Falco, the first step is to determine its placement. Falco processes rules in sequential order, ensuring that two rules are not triggered simultaneously. Consequently, more general rules should be positioned at the end of the rule set, while more specific rules should be placed at the beginning. This arrangement prevents general rules from capturing events that should be handled by more specific rules.
+
+It's worth noting that Falco loads a set of predefined rules by default, followed by any custom rules located in the `/etc/falco/rules.d` directory. This configuration is specified in the [`/etc/falco/falco.yaml`](https://github.com/falcosecurity/falco/blob/master/falco.yaml) file, under the `rules_file:` key, as follows:
+
+```
+rules_file:
+  - /etc/falco/falco_rules.yaml
+  - /etc/falco/falco_rules.local.yaml
+  - /etc/falco/rules.d
+```
+
+It can nevertheless be adjusted to prioritize the rules in the `rules.d` directory, include different rule files, or even add new directories. The customization options are flexible.
+
+Considering this, it's important to be mindful that the default rules file includes reusable lists and macros that may be helpful for creating new rules. Therefore, you should carefully decide whether you want to delay the processing of those files or disable a general rule that captures events intended for your custom rule. Alternatively, you can customize an existing rule within the `rules.d` directory by either rewriting the entire rule or using the "append" key to modify it.
+
+If you are deploying Falco on a Kubernetes cluster, you will likely use [Helm](https://helm.sh) for the installation. In this scenario, instead of placing custom rules files directly in the `/etc/falco/rules.d` directory, you can add them to the `values.yaml` file provided to the `helm` command.
+
+Locate the line `customRules: {}` in the [`values.yaml`](https://github.com/falcosecurity/charts/blob/master/falco/values.yaml) file and replace it with a configuration similar to the following:
+
+```
+customRules:
+  custom-rules.yaml: |-
+    - rule: Example rule
+      desc: ...
+      ...
+    - rule: Example rule 2
+      ...
+  more-custom-rules.yaml: |-
+    - rule: ...
+      ...
+```
+
+That will instruct `helm` to create as many rules files as you define here accessible inside the Falco Pods, under the directory `/etc/falco/rules.d`.
+
+## Rules Structure
+
+Rules in Falco are defined using YAML syntax. Each rule is represented as an element in a list, and this is denoted by using a hyphen - before the first key in the rule. When creating a new rule, there are several essential keys that should be included:
```

Comment:
`tags` are not a requirement to write a rule. I find they add complexity when we talk about writing custom rules since their main purpose is to allow better management of these.
I'll mention though that there are more fields and where to find them.

Reply:
I think this is a great summary, we shouldn't advertise that they are not needed or not meaningful. Also we will introduce updated tags filtering mechanisms, therefore let's consider this topic fluid and to be improved / updated in the future.

Reply:
I believe I managed to address it without going too much into detail.